ReportSimulationOutput/measure.rb

```diff
@@ -1558,18 +1582,23 @@ def write_timeseries_output_results(runner, outputs, output_format,
     else
       weather_data = []
     end
+    if !@output_variables.empty?
+      output_variables_data = @output_variables.values.select { |x| x.timeseries_output.sum(0.0) != 0 }.map { |x| [x.name, x.timeseries_units] + x.timeseries_output.map { |v| v.round(dig) } }
```

Comment:
Since we don't know what kind of units/output is being requested, it's probably a bad idea to round the value. I suggest we just report what EnergyPlus gives us.

```suggestion
      output_variables_data = @output_variables.values.select { |x| x.timeseries_output.sum(0.0) != 0 }.map { |x| [x.name, x.timeseries_units] + x.timeseries_output }
```